server/user.go

```diff
@@ -311,21 +311,23 @@ func (p *Plugin) disconnectUser(instance Instance, user *User) (*Connection, err
 	return conn, nil
 }
 
-func (p *Plugin) GetJiraUserFromMentions(instanceID types.ID, mentions model.UserMentionMap, userKey string) *jira.User {
+func (p *Plugin) GetJiraUserFromMentions(
+	instanceID types.ID, mentions model.UserMentionMap, userKey string,
+) (*jira.User, error) {
```

Comment:
Putting this on multiple lines isn't necessarily common practice in our projects. It took me a second to realize what's going on here. What are your thoughts on having it formatted how it was before?

Reply:
If that's the standard, I'm fine with it. :-)

I try not to pass the 80 character line limit, because then when I split the screen I can't see some code, but it's ok, I'll change it!

Thank you! :-)